### PR TITLE
update policy document

### PR DESCRIPTION
**Pull request by akash1810**
As noted in https://github.com/guardian/domains-platform/pull/52/files#r512522245, update policy document.

This PR also adds a config file for EditorConfig to encourage consistency of intents.

Diff is best seen [w/out whitespace](https://github.com/guardian/ssm-scala/pull/124/files?diff=split&w=1).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -463,16 +463,20 @@ To use ssm-scala against the instances of your project, the following needs to h
...
+                  - ec2messages:GetMessages
+                  - ec2messages:SendReply
+                  - ssm:UpdateInstanceInformation
+                  - ssm:ListInstanceAssociations
+                  - ssm:DescribeInstanceProperties
+                  - ssm:DescribeDocumentParameters
+                  - ssmmessages:CreateControlChannel
+                  - ssmmessages:CreateDataChannel
+                  - ssmmessages:OpenControlChannel
+                  - ssmmessages:OpenDataChannel
```
Comment (jamie-lynch) [inline README.md:479]: I think these should all be indented the same?

Comment (akash1810) [reply README.md]: Yep good spot. Updated now via an IDE 😄

Comment (jamie-lynch) [reply README.md]: It looks differently wrong to me now 😕

Comment (akash1810) [reply README.md]: I don't know why GH is rendering it so strangely!

**VS Code**
image

**GitHub**
image

Comment (akash1810) [reply README.md]: Oh, what have I done?! 😱

Comment (akash1810) [reply README.md]: Ah, there was a mixture of tabs and spaces being used. I've converted to spaces now and its looking better.

Comment (jamie-lynch) [reply README.md]: Awesome! 👍

Comment (akash1810) [reply README.md]: Added config for [EditorConfig](https://editorconfig.org/) to this project now, to prevent this dance in the future 🕺🏽 .

**Review by jamie-lynch — APPROVED** [review]

**Pull request merged**